acceptance/openstack/loadbalancer/v2/loadbalancers_test.go

```diff
@@ -101,7 +101,7 @@ func TestLoadbalancersCRUD(t *testing.T) {
 	tools.PrintResource(t, newListener)
 
 	// L7 policy
-	policy, err := CreateL7Policy(t, client, listener, lb)
+	_, err := CreateL7Policy(t, client, listener, lb)
```

Comment:
I'm so sorry, this line should be:

```
_, err = CreateL7Policy(t, lbClient, listener, lb)
```

Note the omission of the `:` since there are no new variables being declared.

But also note that `client` needs changed to `lbClient` as that was changed during the move to the `octavia` directory.

Reply:
It should be me to say sorry for my carelessness, thanks for the review!